src/minecraftServers/Controllers/ServersController.cs

```diff
@@ -65,7 +65,7 @@ public ActionResult Update(ServerDto server, uint id)
         }
         var updatedServer = new Server(server.Ip, exsistedServer.Online, server.Name, id);
         _serversRep.Delete(exsistedServer.Id);
-        _serversRep.AddUnique(updatedServer, id);
+        _serversRep.Add(updatedServer);
         return Ok(server);
```

Comment:
I think it's better to  actual update an entity instead of removing it and adding a new one